docs/book/src/jobs.md

```diff
@@ -0,0 +1,42 @@
+# Jobs
+
+This document intents to provide an overview over our jobs running via Prow and Github actions.
+
+## Build and Tests running on the default branch
+
+
+ > NOTE : To see which test jobs execute which e2e tests, you can click on the links which lead to the
+  respective test overviews in testgrid.
+
+### Presubmits
+
+  Prow Presubmits:
+
+  - [pull-cluster-api-provider-azure-build] `./scripts/ci-build.sh` REQUIRED
+  - [pull-cluster-api-provider-azure-test] `./scripts/ci-test.sh`   REQUIRED
+  - [pull-cluster-api-provider-azure-e2e] `./scripts/ci-e2e.sh`     REQUIRED
+  - [pull-cluster-api-provider-azure-e2e-exp] `./scripts/ci-e2e.sh`
+  - [pull-cluster-api-provider-azure-windows] `./scripts/ci-e2e.sh` REQUIRED
+  - [pull-cluster-api-provider-azure-verify]                        REQUIRED
+  - [pull-cluster-api-provider-azure-apidiff] `./scripts/ci-apidiff.sh`
+  - [pull-cluster-api-provider-azure-coverage] `./scripts/ci-test-coverage`
+  - [pull-cluster-api-provider-e2e-full] 
+  - [pull-cluster-api-provider-capi-e2e]
+  - [pull-cluster-api-provider-azure-conformance-v1alpha4]
+  - [pull-cluster-api-provider-azure-upstream-v1alpha4-windows]
+  - [pull-cluster-api-provider-azure-conformance-with-ci-artifacts]
+  - [pull-cluster-api-provider-azure-windows-upstream-with-ci-artifacts]
+
+  Github Presubmits Workflows:
+
+
+
+### Postsubmits
+   
+  - [post-cluster-api-provider-azure-push-images] `/run.sh`
+  - [postsubmits-cluster-api-provider-azure-e2e-full-main] `runner.sh` `./scripts/ci-e2e.sh`
+
+### Periodics
```

Comment:
let's focus on presubmits for now, we can add periodics in a part 2

presubmits are the most valuable to understand for new developers